### PR TITLE
LPS-187423 [Martketplace] Fixing the role permissions checks

### PR DESCRIPTION
**Pull request by ytuik**
https://issues.liferay.com/browse/LPS-187423

### CODE REVIEW

**Comment by ytuik** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboardPage/PurchasedDashboardPageUtil.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const customerPermissionDescriptions : PermissionDescription[] = [
   {
     permissionName: 'Purchase apps and solutions',
     permissionTooltip: 'Purchase new apps and versions',
-    permitedRoles: ['Account Administrator', 'Account Buyer'],
+    permitedRoles: ['Account Buyer'],
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboardPage/PurchasedDashboardPageUtil.ts:59]: Please, change it here and other parts of the code.
```suggestion
    permittedRoles: ['Account Buyer'],
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions.

**Review by silasbsi — APPROVED** [review]
Great job

**Comment by ytuik** [comment]
ci:forward

**Pull request closed without merging**